Compiler/ThriftCompiler.php

```diff
@@ -97,6 +103,15 @@ public function setModelPath($path)
     }
 
     /**
+     * Add a directory to the list of directories searched for include directives
+     * @param string[] $includeDirs
+     */
+    public function setIncludeDirs($includeDirs)
+    {
+        $this->includeDirs = (array)$includeDirs;
```

Comment:
This is not a conversion in order to fix values from configuration, but rather a guard from direct user call to the method.